_pages/homepage-coming-soon.html

```diff
@@ -311,6 +354,27 @@ <h2 class="conf-description">Join us for five days of inspiration, education, an
       <h2 class="conf-description">Book your <a href="https://www.marriott.com/events/start.mi?id=1674740649947&key=GRP">hotel room</a> at the <a href="https://www.marriott.com/en-us/hotels/rducv-durham-marriott-city-center/overview/">Marriott Durham City Center</a> by September 16!</h2>
       <p>The hotel connects directly to the convention center.</p>
       <h2 class="conf-description">Our <a href="/speaking/">Call for Proposals</a> and <a href="/opportunity-grants/">Opportunity Grant applications</a> are now open!</h2>
+
+      <br>
+      <br>
+
+      <h2 class="conf-locdate">Latest News</h2>
+      <div class="news">
+        {% for post in site.posts limit:12 %}
+          {% unless post.hidden %}
+          <article class="news-title medium-4 column">
+            <div class="card {% if post.category == "Sponsorship" %}card-theme-brand-color3{% else %}card-theme-brand-color2{% endif %}">
+              <div class="card-section">
+                <span class="post-meta">{{ post.date | date: "%b %-d, %Y" }}</span>
+                <h3>
+                  <a href="{{ post.url | prepend: site.baseurl }}">{{ post.title | escape }}</a>
```

Comment:
Is a relative link not good enough? (In other words, does the prepend need to be there?)

Reply:
Seems like it's fine either way. We have two prior usages that match `{{ post.url | prepend: site.baseurl }}` and then one that uses `{{ post.url }}` (/speaking/status/). I confirmed the later is working on our 2022 site. All to say, I don't think it matters beyond preference.

Reply:
This is copied from our default homepage blocks from previous years. I can change it, but I'm unsure if it matters unless I miss the finer point. 

https://github.com/djangocon/2023.djangocon.us/blob/main/_layouts/home.html#L149